File: python/bc2061.py
cVal = input().split(" ")

nAb = int(cVal[0])
aP = int(cVal[1])

for c in range(0, aP):
    aIn = input()

    nAb -= 1

    if aIn == "fechou": nAb += 2

print(nAb)

File: cpp/bc2061.cpp
#include <iostream>

using namespace std;

int main()
{
	int nAb, aP;
	string aIn;
	
	cin >> nAb >> aP;
	
	for(int i = 0; i < aP; i++)
	{
		cin >> aIn;
		
		nAb -= 1;
		
		if(aIn == "fechou") nAb += 2;
	}
	
	cout << nAb << endl;
}
